programmatic_execution_of_notebooks/description.md

```diff
@@ -0,0 +1,220 @@
+## Programmatic execution of notebooks
```

Comment:
Could you please rename this file to `workspace_dashboard.md` which is what we have in our workspace best practices wiki?